source/usage-examples/changeStream.txt

```diff
@@ -127,8 +127,7 @@ callback process the event before exiting.
    The JavaScript and TypeScript code snippets above are identical. There are no
    TypeScript specific features of the driver relevant to this use case.
 
-If you run the example above, you should see output similar to the
-following:
+If you run the preceding example, you should see output that looks like this:
```

Comment:
I think the style guide "[this](https://docs.mongodb.com/meta/style-guide/writing/use-pronouns-carefully/#this)" guideline is only for beginning a sentence? 

>  Avoid beginning a sentence with the pronoun this, unless you follow this with a noun to clarify its meaning.

[Here is the initial thread ](https://github.com/mongodb/docs-java/pull/57#discussion_r579507100) where we arrived at the "looks like this" style for examples. I don't have a preference either way, but this phrasing is used a fair bit throughout the Java Driver documentation, so would like to make sure this phrasing is outside guidelines before accepting note as it could alter the default format for how we discuss output.

Reply:
@biniona-mongodb, the style guide does note the beginning of a sentence, but it creates the same issues at the beginning or the end of a sentence. Using just "the following" as noted in the other thread creates the same issues as well. From what I understand, it's not a great experience for someone using a screen reader and can create machine translation issues down the line. I'll provide this feedback on the other thread as well.

Reply:
I'm convinced enough to make the edits! I like your proposed phrasing. Hopefully we will all arrive at a consensus on what the best way to say "your output is the next thing on this page". I agree with you at present  that "you should see the following output" seems to be stronger than "looks like this" though.